runtime/builtin_types.go

```diff
@@ -190,6 +190,29 @@ func builtinAbs(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
 	return Abs(f, args[0])
 }
 
+func builtinAll(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+	if raised := checkFunctionArgs(f, "all", args, ObjectType); raised != nil {
+		return nil, raised
+	}
+	iter, raised := Iter(f, args[0])
+	if raised != nil {
+		return nil, raised
+	}
+	o, raised := Next(f, iter)
+	for ; raised == nil; o, raised = Next(f, iter) {
+		if ret, raised := IsTrue(f, o); raised != nil {
+			return nil, raised
+		} else if !ret {
```

Comment:
Thanks for pointing that out. The code I wrote isn't correct for that reason. But declaring outside the if is the style I've followed in the source code. It is more verbose (and also pollutes the outer scope's namespace more) but I think the clarity is worth it.